packages/mdc-select/README.md

```diff
@@ -293,6 +297,10 @@ within `componentDidUpdate`.
 
 ### The full foundation API
 
+#### MDCSelectFoundation.getValue() => string
+
+Returns the value of the currently selected option.
```

Comment:
Maybe also document when invalid what happens if user haven't made any choice.

"Returns the value of the currently selected option. If none of options is selected yet, return empty string."